src/contexts/Help/types.ts

```diff
@@ -4,23 +4,35 @@
 import { ReactNode } from 'react';
 import { MaybeString } from 'types';
 
-export type HelpContentRaw = Array<HelpItemRaw>;
+export type HelpItemLocales = Array<HelpItemLocale>;
```

Comment:
What was the reasoning for changing these names?

Reply:
Just more intuitive for me